pkg/routeagent_driver/main.go

```diff
@@ -155,8 +158,20 @@ func main() {
 		mtu.NewMTUHandler(env.ClusterCidr, len(env.GlobalCidr) != 0, getTCPMssValue(localNode)),
 		calico.NewCalicoIPPoolHandler(cfg, env.Namespace, k8sClientSet),
 		healthchecker.New(healthcheckerConfig,
-			smClientset.SubmarinerV1().RouteAgents(submSpec.Namespace), versions.Submariner(), localNode.Name))
+			smClientset.SubmarinerV1().RouteAgents(submSpec.Namespace), versions.Submariner(), localNode.Name),
+	}
+
+	globalConfigMap, err := k8sClientSet.CoreV1().ConfigMaps(env.Namespace).Get(ctx, globalConfigMapName, metav1.GetOptions{})
+	enableAvoidSNAT, err := ovn.NeedToEnableAvoidSNATHandler(globalConfigMap)
+
+	if err == nil && enableAvoidSNAT {
```

Comment:
Since this is a user error we shouldn't silence it.

```suggestion
    logger.FatalOnError(err, "Error determining avoid SNAT enablement")
    
	if err == nil && enableAvoidSNAT {
```